pkg/kn/commands/service/create.go

```diff
@@ -208,7 +209,7 @@ func prepareAndUpdateService(ctx context.Context, client clientservingv1.KnServi
 		changed, err := client.UpdateService(ctx, service)
 		if err != nil {
 			// Retry to update when a resource version conflict exists
-			if apierrors.IsConflict(err) && retries < MaxUpdateRetries {
+			if apierrors.IsConflict(err) && retries < config.DefaultRetry.Steps {
```

Comment:
I know this's only changed due to removal of variable, but looking at the code itself I wonder if shouldn't be just `.UpdateServiceWithRetry` without that additional conflict handling block.